src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,24 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveArtworkItem: {
+      type: ConversationItemType,
+      resolve: (artworkItem, _args, { artworkLoader }) => {
+        if (artworkItem.item_type === "Artwork") {
+          return artworkLoader(artworkItem.properties.id).then((artwork) => {
+            const updatedArtwork = {
+              ...artwork,
+              __typename: "Artwork",
+            }
+            return artwork.published ? updatedArtwork : null
```

Comment:
Added a check to make sure the artwork is published here per @joeyAghion [comment](https://github.com/artsy/metaphysics/pull/3249#issuecomment-873189728).

Reply:
This wasn't precisely what I was asking about (or maybe I just don't follow).

If the artwork isn't published, this resolver will _fail_ because it's not authorized to load the artwork's data. Is that expected? (And will clients tolerate it?) The intent of the snapshot was to represent the subset of artwork data that _could_ be exposed long after an artwork is updated or unpublished.

Reply:
Ah I understand what you're saying now. No I didn't realize that was the case but I think it's something we could maybe handle on the client.

Reply:
The goal is to fetch "live" artwork data as part of the same conversation query, instead of the snapshot, so we can render the "Make Offer" button properly. For example, an artwork can be updated to sold after a conversation is made.

If the live artwork becomes unavailable (e.g. unpublished or deleted), it makes sense that this field raises an exception (or returns null?) and clients need to handle it. I'm not entirely positive what the response looks like in this case, so we should test it and make sure clients handle it properly.

@xtina-starr let me know if you want to pair on this together!